### DISCUSSION TITLE
Spell Checker Info panel - what is it for?

### DISCUSSION THREAD

**Discussion by JessePeden** [streetsidesoftware/vscode-spell-checker#3811] [Q&A] [answered]
What is the purpose of the "Spell Checker Info" panel that was added during one of the recent updates?  It shows nothing and I can't find any documentation about what it's for or why (or exactly when) it was added, or why, if it was always there, it only began showing the icon in a recent update.

**Comment by Jason3S** [comment, marked as answer]
The info panel got broken when updating the UI framework from Svelte 4 to Svelte 5.

The purpose of the Info Panel is to display information relevant to the file, like:
- file type
- enabled / disabled for spell checking
- dictionaries used
- excluded by glob or configuration.

It had links to the settings / configuration impacting how a file is spell checked.

I need to look at why it got broken when the Svelte library was updated.

**Reply by JessePeden** [reply]
Ah.  That makes sense, then.  Thanks for the info.

**Comment by Jason3S** [comment]
@JessePeden,

It has been updated. Do you have any feedback on the info?
